UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ValidateAuthorizationForAPIsWithExchangeSetStandardParameterTests.cs

```diff
@@ -23,102 +22,79 @@ public void SetupAsync()
         // PBI 140109 : ESS API : Add authorization to allow only UKHO people to create unencrypted ES 
         [Test]
         [Category("SmokeTest-AIODisabled")]
-        public async Task WhenICallTheSinceDateTimeApiWithAValidB2cTokenAndIsUnencryptedParameterAsTrue_ThenAnUnauthorizedResponseIsReturned()
+        public async Task WhenICallSinceDateTimeApiWithAValidB2cTokenAnds57AsexchangeSetStandardParameter_ThenAnUnauthorizedResponseIsReturned()
```

Comment:
Please change wherever applicable in FTs
```suggestion
        public async Task WhenICallSinceDateTimeApiWithAValidB2cTokenAndS57AsExchangeSetStandardParameter_ThenAnUnauthorizedResponseIsReturned()
```

Reply:
Resolved. Thanks